src/R3.WinUI3/R3.WinUI3.csproj

```diff
@@ -1,14 +1,32 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Uno.Sdk">
     <PropertyGroup>
-        <TargetFramework>net6.0-windows10.0.19041.0</TargetFramework>
-        <TargetPlatformMinVersion>10.0.17763.0</TargetPlatformMinVersion>
+        <DotNetVersion>net8.0</DotNetVersion>
+        <TargetFrameworks Condition="$([MSBuild]::IsOSPlatform('windows')) or '$(EnableWindowsTargeting)' == 'true'">$(TargetFrameworks);$(DotNetVersion)-windows10.0.22621</TargetFrameworks>
+        <TargetFrameworks>$(TargetFrameworks);$(DotNetVersion)</TargetFrameworks>
```

Comment:
@neuecc 
You can see here that it adds `net8.0;net8.0-windows10.x.xxxxx`
This means that the NuGet package will have code for WinUI and for all Uno-supported platforms